src/Microsoft.AspNetCore.Antiforgery/Internal/DefaultAntiforgeryTokenStore.cs

```diff
@@ -69,9 +69,13 @@ public void SaveCookieToken(HttpContext httpContext, string token)
             Debug.Assert(httpContext != null);
             Debug.Assert(token != null);
 
-            var options = new CookieOptions();
-            options.HttpOnly = true;
-            options.Domain = _options.CookieDomain;
+            var options = new CookieOptions
+            {
+                HttpOnly = true,
+                Domain = _options.CookieDomain,
+                SameSite = SameSiteEnforcementMode.Strict
```

Comment:
I am not familiar with this option, Let's have a conversation with @blowdart - we need to understand what should be the default and **if** this should be configurable.